src/common/post.ts

```diff
@@ -184,6 +187,23 @@ export const createFreeformPost = async (
       private: privacy,
     },
   });
+
+  const vordrStatus = await checkWithVordr(
+    { post: createdPost },
+    { con, userId: ctx.userId, req: ctx.req },
+  );
+
+  if (vordrStatus === true) {
+    createdPost.banned = true;
+    createdPost.flags.banned = true;
+
+    createdPost.showOnFeed = false;
+    createdPost.flags.showOnFeed = false;
```

Comment:
You might want to use `updateFlagsStatement` here for safety :)

Reply:
I get this TS error then

> Type '() => string' has no properties in common with type 'Partial<{ sentAnalyticsReport: boolean; banned: boolean; deleted: boolean; private: boolean; visible: boolean; showOnFeed: boolean; promoteToPublic: number; deletedBy: string; vordr: boolean; }>'.ts(2559)

I think it should be fine to not use it, as it hasn't been saved in the database yet, so it's still a js object.

Reply:
Can do this instead
```ts
createdPost.flags = {
  ...createdPost.flags,
  banned: true,
  showOnFeed: false,
};
```

Reply:
CC: @capJavert up to you :)

Reply:
@omBratteng for `updateFlagsStatement` you have to use insert or update methods since save does not support sql statements, since we are just creating a post we can probably use `insert` here.

Reply:
But I think we don't have to use `updateFlagsStatement` here since as @omBratteng said it has not been saved to DB yet.

Reply:
@capJavert went with this as it's before insertion to DB.
```ts
createdPost.flags = {
  ...createdPost.flags,
  banned: true,
  showOnFeed: false,
};
```